python-sdk/nuscenes/eval/panoptic/panoptic_track_evaluator.py

```diff
@@ -129,6 +133,38 @@ def get_panoptic_track_stats(self,
             unique_combo_, counts_combo_ = np.unique(offset_combo_, return_counts=True)
             self.update_dict_stat(cl_intersects, unique_combo_, counts_combo_)
 
+            # Computation for PAT score
+            unique_gt_, counts_gt_ = np.unique(y_inst_in_cl[y_inst_in_cl > 0], return_counts=True)
+            id2idx_gt_ = {inst_id: idx for idx, inst_id in enumerate(unique_gt_)}
+            unique_pred_, counts_pred_ = np.unique(x_inst_row[x_inst_row > 0], return_counts=True)
+            id2idx_pred_ = {inst_id: idx for idx, inst_id in enumerate(unique_pred_)}
+            gt_labels_ = unique_combo_ // self.offset
+            pred_labels_ = unique_combo_ % self.offset
```

Comment:
Perhaps some comments on what these two lines are doing?:
```
gt_labels_ = unique_combo_ // self.offset
pred_labels_ = unique_combo_ % self.offset
```

Reply:
Added the comment. Let me know if it is intuitive.